app/src/main/AndroidManifest.xml

```diff
@@ -16,6 +16,10 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <meta-data
+            android:name="preloaded_fonts"
+            android:resource="@array/preloaded_fonts" />
+
```

Comment:
Genuine question, what does this do?

Reply:
I think [this guide](https://developer.android.com/guide/topics/ui/look-and-feel/downloadable-fonts#predeclaring-fonts) helps.

Reply:
Thanks!